library/errata_tool_product_version.py

```diff
@@ -255,11 +288,11 @@ def prepare_diff_data(before, after):
         after=after,
         item_name=after['name'],
         item_type='product version',
-        keys_to_copy=[
-            # This field exists in ET but is not yet supported by
-            # this ansible module
-            'use_quay_for_containers',
-        ],
+        # keys_to_copy=[
```

Comment:
Please delete this altogether instead of commenting it out.

Reply:
Done.